### DISCUSSION TITLE
能否更新一下app icon

### DISCUSSION THREAD

**Discussion by qinyihao** [cpeditor/cpeditor-cn#1] [想法]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is this feature provided in the master branch?

- [X] I have checked the changelog

### The problem you are facing

cpeditor的图标在launchpad里面比较不和谐

### Describe the feature you'd like

能否加入如下图片作为icon：
![photo](https://user-images.githubusercontent.com/40647091/123257017-346c8c00-d524-11eb-8f2b-7e3565d458d7.png)

### Are you willing to contribute to this?

- [ ] I'm ready to contribute to this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [X] No, thanks

**Comment by qinyihao** [comment]
原本是 https://github.com/cpeditor/cpeditor/issues/890

**Comment by ouuan** [comment]
你说的 launchpad 是啥

**Comment by swiftqwq** [comment]
<img width="306" alt="Screen Shot 2021-06-25 at 6 07 10 PM" src="https://user-images.githubusercontent.com/61449077/123408950-3562f380-d5e0-11eb-9714-a56712b2c76b.png">

感觉也没有不协调啊（

**Comment by qinyihao** [comment]
启动台（mac的

在 2021-06-24 23:50:04，"Yufan You" ***@***.***> 写道：


你说的 launchpad 是啥

**Comment by ouuan** [comment]
你用的什么版本？是 6.9.2 吗？

**Reply by qinyihao** [reply]
是的

**Comment by ouuan** [comment]
能说明一下现在的 icon 和你给的这个有什么区别吗

**Comment by qinyihao** [comment]
这个6.8.2的icon还是原来的那个完全圆形的icon啊（难道这icon没更新？



在 2021-06-26 17:22:57，"Yufan You" ***@***.***> 写道：

能说明一下现在的 icon 和你给的这个有什么区别吗

**Reply by ouuan** [reply]
你说你在用 6.9.2: https://github.com/cpeditor/cpeditor-cn/discussions/1#discussioncomment-923773

你在开 issue 的时候还说你检查了 master branch 和 changelog...

**Comment by qinyihao** [comment]
az
那么求解这个icon不更新的原因（



在 2021-06-26 18:34:42，"Yufan You" ***@***.***> 写道：

你说你在用 6.9.2: #1 (reply in thread)

你在开 issue 的时候还说你检查了 master branch 和 changelog...

**Reply by ouuan** [reply]
> 那么求解这个icon不更新的原因（

你说哪个版本？

**Comment by qinyihao** [comment]
6.9.2
sys ver: macOS 12.0 Developer Beta 2



在 2021-06-26 21:49:51，"Yufan You" ***@***.***> 写道：

那么求解这个icon不更新的原因（

你说哪个版本？

**Reply by ouuan** [reply]
6.9.2 的 icon 和你发的 icon 有什么区别？你能用一句完整的、不带代词的话来表述你的需求吗？我不明白你什么意思，因为 6.9.2 的 icon 和你发的这个（几乎）是一样的。
